docs/reference/module-overview.md

```diff
@@ -120,13 +122,13 @@ modules.export = {
 
 #### `group`
 
-An object of field groups. The groupings are used by editing interface. It _does not apply to widget modules_, which do not have a tabbed interface.
+An object of field groups. Groupings are used by editing interface. Note that `group` _does not apply to widget modules_.
 
 Groups are added as an object with their name as the object key and the following properties:
-- `label`: The visible label (a string) on the interface tab
+- `label`: The visible label (a string) for the group
 - `fields`: An array of field names to include in the group
 
-The `@apostrophecms/doc-type` module groups the default fields in a `basics` group, which you can override. If overriding the `basics` group, you will need to add those core fields to a group in the module. Any fields not added to a group will be placed in an "Ungrouped" section in the editing interface.
+The `@apostrophecms/doc-type` module arranges the default fields in a `basics` group. You can override the `basics` group, but those default fields will become ungrouped unless you arrange them again. Any fields not added to a group will be placed in an "Ungrouped" section in the editing interface.
 
```

Comment:
What about utility?

Reply:
... off limits. Just kidding, but maybe we shouldn't allow utility fields to be moved? Or to be ungrouped? @localghost443 ?

Reply:
I think it's best to leave it up to re-grouping like anything else, since we can't anticipate all future situations and our extensibility is here to accommodate them, but it's fair to say it would be best practice to leave the standard utility fields in that group when they exist.

Reply:
I tend to favor the idea hat utility fields shouldn't be moved. I don't think it offers any utility and adds complexity. Its definitely best practice not to move them. Some powers shouldn't be given or encouraged.

Reply:
Seems like a good idea to me. That should be a new ticket for "some day"